packages/flame_texturepacker/lib/atlas/model/texture_packer_sprite.dart

```diff
@@ -32,7 +32,7 @@ class TexturePackerSprite extends Sprite {
         ) {
     decorator = Transform2DDecorator(transform);
     if (region.rotate) {
-      transform.angle = radians(90);
+      transform.angle = radians(region.degrees.toDouble());
```

Comment:
The `TexturePackerSprite` has a getter for the radians called `angle`, maybe add that getter to the region too?
https://github.com/flame-engine/flame/pull/2987/files#diff-f5ec3095fee21654a4f4b11da4431af1fb3e668960a0c9f7f53b5ff239d15d78R83